src/graphs/block_vertex_tree.rs

```diff
@@ -0,0 +1,60 @@
+//! # Block Vertex Tree
+
+/// # Guarantees
+/// - 0 <= bcc_id\[u\] < num_bccs
+///
+/// # Example
+/// ```
+/// use programming_team_code_rust::graphs::cuts::get_cuts;
+/// use programming_team_code_rust::graphs::block_vertex_tree::get_bvt;
+///
+/// let edge_list = [(0,1), (0,1), (1,2), (0,1)];
```

Comment:
I think we should start initializing our graphs like this even though it's more code

because edge list is easier to imagine what the graph would be

Reply:
I agree, good point

Reply:
can we change the other ones in this diff as well?